db/db.ts

```diff
@@ -18,30 +18,40 @@ try {
     throw error
 }
 
-async function getTenantConfig(tenant: string): Promise<any> {
+async function getTenantConfig(tenant: string, driver: string): Promise<any> {
```

Comment:
I think the return type could be :  

```ts
Promise<TenantConfig | null>
```

based on its uses in api.ts

Reply:
You're right. Working on it

Reply:
Done